test/examps/LuckyPeople/TestLuckyPeople.bat

```diff
@@ -3,26 +3,38 @@
 @REM See LICENSE at root directory of this repository.
 
 @ECHO OFF
-REM where torxakis.bat
 SET TORXAKISPORT=torxakisPort.bat
 SET PORT=9876
 
 SET ASSIGN=%1\examps\LuckyPeople\sut
 SET SOL=%1\examps\LuckyPeople\spec
 SET TEST=%1\test\examps\LuckyPeople
 
-echo ------- Start LuckyPeople Test 1
+echo ------- Start LuckyPeople Examples Purpose Test
 start /min java -cp %ASSIGN% LuckyPeople
-call %TORXAKISPORT% %PORT% %SOL%\LuckyPeople.txs < %TEST%\LuckyPeopleExamples_Tester.txscmd
+call %TORXAKISPORT% %PORT% %SOL%\LuckyPeople.txs < %TEST%\LuckyPeopleExamples_Purpose.txscmd
 TIMEOUT /T 4 /NOBREAK
-move /Y .txs.%PORT%.err.log Examples.txs.%PORT%.err.log
-move /Y testTraceExamples.log testTrace.Spec.LuckyPeopleExamples.Sut.LuckyPeople.log
-echo ------- End LuckyPeople Test 1
+move /Y .txs.%PORT%.err.log Examples_Purpose.txs.%PORT%.err.log
+echo ------- End LuckyPeople Test Examples Purpose Test
 
-echo ------- Start LuckyPeople Test 2
+echo ------- Start LuckyPeople Random Test
 start /min java -cp %ASSIGN% LuckyPeople
 call %TORXAKISPORT% %PORT% %SOL%\LuckyPeople.txs < %TEST%\LuckyPeopleRandom_Tester.txscmd
 TIMEOUT /T 4 /NOBREAK
 move /Y .txs.%PORT%.err.log Random.txs.%PORT%.err.log
-move /Y testTraceRandom.log testTrace.Spec.LuckyPeopleRandom.Sut.LuckyPeople.log
-echo ------- End LuckyPeople Test 2
\ No newline at end of file
+echo ------- End LuckyPeople Random Test
+
+echo ------- Start LuckyPeople By Gender Test
+start /min java -cp %ASSIGN% LuckyPeople
+call %TORXAKISPORT% %PORT% %SOL%\LuckyPeople.txs < %TEST%\LuckyPeopleByGender_Purpose.txscmd
+TIMEOUT /T 4 /NOBREAK
+move /Y .txs.%PORT%.err.log ByGender.txs.%PORT%.err.log
+echo ------- End LuckyPeople By Gender Test
+
+REM echo ------- Start LuckyPeople Overflow Test
```

Comment:
Why are we commenting this out? Can't we just remove this?

Reply:
I'm not sure about this Overflow test; it requires another 300 steps to trigger byte overflow for both genders (128 per gender, round up to make up for Quiescence's) and current test purpose is not enough to test it - it should ensure that when switched to other gender the new person is not lucky by some other criteria.

I am going to open an issue for this, so I wanted to keep it as reference. We can remove it after we decide what to do about this case.

Reply:
Thinking again, I will just remove it. These lines don't add value even as reference.